src/main/java/com/keniu/services/ShoppingCartService.java

```diff
@@ -0,0 +1,19 @@
+package com.keniu.services;
+
+import com.keniu.dto.CreateCartItemRequestDto;
+import com.keniu.dto.ShoppingCartDto;
+import com.keniu.dto.UpdateCarItemRequestDto;
+import com.keniu.models.ShoppingCart;
+
+public interface ShoppingCartService {
+    
+    ShoppingCart save(ShoppingCart shoppingCart);
+    
+    ShoppingCartDto getShoppingCart(Long userId);
+    
+    ShoppingCartDto addCartItem(Long userId, CreateCartItemRequestDto createCartItemRequestDto);
+    
+    ShoppingCartDto update(Long userId, Long id, UpdateCarItemRequestDto updateCarItemRequestDto);
+    
+    ShoppingCartDto delete(Long userId, Long id);
```

Comment:
The same method in BookService is void type)

Reply:
Why

Reply:
Aaa. Because in this case it's actually updating Cart by deleting some items.